test/torchaudio_unittest/datasets/commonvoice_test.py

```diff
@@ -130,8 +113,8 @@ class TestCommonVoiceFR(TempDirMixin, TorchaudioTestCase):
     @classmethod
     def setUpClass(cls):
         cls.root_dir = cls.get_base_temp_dir()
-        cls.data = get_mock_dataset_fr(cls.root_dir)
         COMMONVOICE._ext_audio = ".mp3"
```

Comment:
oh I think I missed the in the previous PR, but `_ext_audio` should be `.wav` for all the unit test, because `load` function for Windows does not support `mp3`. Yet, underlying PySoundFile checks the header and it knows that it's WAVE format so the test was passing.